package/yast2-country.spec

```diff
@@ -28,7 +28,7 @@ License:        GPL-2.0
 #policy files for YaPI dbus interface
 Source1:        org.opensuse.yast.modules.yapi.time.policy
 Source2:        org.opensuse.yast.modules.yapi.language.policy
-BuildRequires:	perl-XML-Writer update-desktop-files yast2-testsuite yast2 yast2-perl-bindings
+BuildRequires:	perl-XML-Writer update-desktop-files yast2-testsuite yast2-perl-bindings
```

Comment:
I suggest to run spec formatter on this spec as it looks a bit ugly. script is at /usr/lib/obs/service/format_spec_file